lib/backend/model/bgpconfig.go

```diff
@@ -17,13 +17,18 @@ package model
 import (
 	"fmt"
 	"reflect"
+	"regexp"
+
+	log "github.com/Sirupsen/logrus"
 
 	"github.com/projectcalico/libcalico-go/lib/errors"
 )
 
 var (
-	typeGlobalBGPConfig = rawStringType
-	typeHostBGPConfig   = rawStringType
+	matchGlobalBGPConfig = regexp.MustCompile("^/?calico/bgp/v1/global/(.+)$")
```

Comment:
Oh I think it was for "no surprises" later on -- was thinking ahead about moving away from confd and using a common set of library function for both backends to configure bird.  Strictly not required for this PR though.